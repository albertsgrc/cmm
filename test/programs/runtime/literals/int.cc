/*
description{Tests int literal parsing}
*/

#include <iostream>
using namespace std;

void print(int n) { cout << n << endl; }

int main() {
    print(2147483647);
    print(2147483648);
    print(0);
    print(23);
    print(425);
    print(7826);
    print(44594);
    print(283904);
    print(2476268);
    print(42767728);
    print(242782299);
}