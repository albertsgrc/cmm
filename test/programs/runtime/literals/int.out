2147483647
-2147483648
0
23
425
7826
44594
283904
2476268
42767728
242782299
